SimFastTiming/FastTimingCommon/src/BTLPulseShape.cc

```diff
@@ -0,0 +1,3123 @@
+#include "SimFastTiming/FastTimingCommon/interface/BTLPulseShape.h"
+
+
+BTLPulseShape::~BTLPulseShape() { }
+
+
+BTLPulseShape::BTLPulseShape() :
+  MTDShapeBase()
+{
+  buildMe() ;
+}
+
+
+void BTLPulseShape::fillShape( MTDShapeBase::DVec& aVec ) const
+{
+
+  unsigned int index = 0;
+
+  // --- Pulse shape for a signal of 100 p.e.
+
+  aVec[index++] = 1.098909e-08; // time = 0.00 ns (index=0)
```

Comment:
just initialize the vector directly to this list of values instead of stepping through one at a time and incrementing an index